src/main/java/frc/robot/subsystems/indexer/Indexer.kt

```diff
@@ -14,15 +14,22 @@ object Indexer : Subsystem {
             return inputs.beamBroken
         }
 
+    object AutoIndexCommand: SequentialCommandGroup (
```

Comment:
This should really be a class, not an object. It would be easy to accidentally schedule it twice.